src/Microsoft.Health.Dicom.SqlServer/Features/Query/SqlQueryGenerator.cs

```diff
@@ -121,6 +132,85 @@ private void AppendFilterTable(string filterAlias)
             _stringBuilder.AppendLine($") {filterAlias}");
         }
 
+        private void AppendCustomTagTables()
+        {
+            Dictionary<CustomTagDataType, CustomTagLevel> queriedCustomTagDataTypesPerLevel = new Dictionary<CustomTagDataType, CustomTagLevel>();
```

Comment:
Where is this data used? I don't see it being used in the SQL generation

Reply:
This is old code, that I removed.

---
In reply to: [582212411](https://github.com/microsoft/dicom-server/pull/567#discussion_r582212411) [](ancestors = 582212411)